src/SelfService/Application/InvitationApplicationService.cs

```diff
@@ -140,13 +140,13 @@ UserId createdBy
         Capability capability
     )
     {
-        var description = $"\"{inviter}\" has invited you to join capability \"{capability.Description}\"";
+        var description = $"\"{inviter}\" has invited you to join capability \"{capability.Name}\"";
```

Comment:
Are you sure we want this?
Description is very much less human readable.

I agree this text is bad, but I fear showing the name itself may be confusing 🤷